assets/pages/contributions-landing/components/contributionsBundle.jsx

```diff
@@ -69,13 +84,14 @@ const ctaLinks = {
 // ----- Functions ----- //
 
 const getContribAttrs = ({
-  contribType, contribAmount, intCmp, showMonthly,
+  contribType, contribAmount, intCmp, showMonthly, isoCountry,
```

Comment:
Don't think this is [possible](https://github.com/facebook/flow/issues/235) within object restructuring.